frappe/model/naming.py

```diff
@@ -24,42 +24,45 @@ def set_new_name(doc):
 
 	doc.run_method("before_naming")
 
-	autoname = frappe.get_meta(doc.doctype).autoname or ""
+	if draft_name:
+		doc.name = "({0})".format(make_autoname('hash', doc.doctype))
+	else:
+		autoname = frappe.get_meta(doc.doctype).autoname or ""
 
-	if autoname.lower() != "prompt" and not frappe.flags.in_import:
-		doc.name = None
+		if autoname.lower() != "prompt" and not frappe.flags.in_import:
+			doc.name = None
 
-	if getattr(doc, "amended_from", None):
-		_set_amended_name(doc)
-		return
+		if getattr(doc, "amended_from", None):
+			doc.name = _get_amended_name(doc)
```

Comment:
This is causing failure. There's no such function in the diff or full codebase 😄 

@fproldan

Reply:
Fixed